src/loaders/earthEngineLoader.js

```diff
@@ -38,27 +39,32 @@ const earthEngineLoader = async config => {
                     })
                     .then(toGeoJson);
 
+                if (!associatedGeometries.length) {
+                    alerts.push({
+                        warning: true,
+                        message: `${coordinateField.name}: ${i18n.t(
```

Comment:
It might be possible to use a one-time ns separator so that this can be in one string (e.g. semicolon in this example):
```
message: i18n.t(`{{name}}: No coordinates found`, {name: coordinateField.name, nsSeparator: ';'})
```

Reply:
Changed!